.github/workflows/_ember-exam.yml

```
@@ -0,0 +1,24 @@
+name: Ember Exam
+
+on:
+  workflow_call:
+    inputs:
+      partition:
+        required: true
+        type: number
+      split:
+        required: true
+        type: number
+
+jobs:
+  exam:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - name: yarn install
+        run: cd ui && yarn install --frozen-lockfile
+      - name: Ember tests
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+        run: mkdir -p /tmp/test-reports && cd ui && yarn exam:parallel -- --split=${{ inputs.split }} --partition=${{ inputs.partition }}
```

I get a little lost in the --sauce-- here...

`yarn exam:parallel` kinda expands to `percy exec --parallel -- ember exam` then these extra args get tacked on to that, right? so we end up with `percy exec --parallel -- ember exam -- --split=1 --partition=4`, right?  the double-double-dash has me wanting to make fewer leaps to know which program ends up with which arguments.

maybe that's okay, but I wanted to mention it in case something nice comes easily to mind for ya.

but aside from that, while we're renovating, maybe this is a little easier to read?

```suggestion
        working-directory: ui
        run: |-
          mkdir -p /tmp/test-reports
          yarn exam:parallel -- --split=${{ inputs.split }} --partition=${{ inputs.partition }}
```

---

It used to be that the intermediate `--` separated flags for the `yarn` command and the flags for the command the `yarn` command ran.

Turns out it's unnecessary now, so I'm updating this to just `yarn exam:parallel --split=${{ .. }} --partition=${{ .. }}`.

And since we don't need that test-reports dir anymore, I think this will look tidy enough on a single line.